cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -2016,10 +2115,12 @@ search = `cs_o365` sourcetype="ms:o365:reporting:messagetrace" _index_earliest=-
     | eval average_emails_per_7days=round(average_emails_per_7days,2) ] \
 | eval PercentIncrease = (count - average_emails_per_7days) / count * 100 \
 | where count > average_emails_per_7days AND PercentIncrease > `cs_email_increase_over_baseline_limit_in_percentage` AND count > 10 \
+| eval cyences_severity = case(count>10*average_emails_per_7days, "critical", count>5*average_emails_per_7days, "high", count>2*average_emails_per_7days, "medium", true(), "low") \
```

Comment:
This alert is specific to Office 365 but the alert name suggests a generic naming convention.
Let's discuss this.